generators/cmd/cli/cmd/root.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var RootCmd = &cobra.Command{
```

Comment:
We should leave things unexported as much as possible unless there's a struct need. I don't think there is one here, so we can go ahead and unexport this.